### PR TITLE
Emoji Review Section Addition in feedback

### LINKED ISSUE #1954: Emoji Review Section Addition in feedback
image

image

Emoji Review Section Addition in feedback

### PR DESCRIPTION
**Pull request by Damini2004**
## Fixes Issue

Close #1954

## Changes proposed

Emoji Review Section Addition in feedback

## Screenshots

Screenshot 2024-07-23 210506

## Note to reviewers

Can you increase the Level.

## Summary by CodeRabbit

- **New Features**
	- Introduced a new Rating Component in the Feedback form, allowing users to rate their experience using interactive emoji buttons.
	- Enhanced visual feedback for rating selection with updated button styles, including GIF representations for ratings.

- **Dependency Updates**
	- Downgraded the `sweetalert2` package version to improve compatibility with other dependencies.

### CODE REVIEW

**Comment by Damini2004** [comment]
image
Replace it with animated emoji kindly review it

**Comment by Damini2004** [comment]
@codervivek5  review it

File: src/User/components/FeedbackForm/Feedback.jsx [hunk 1]
```diff
@@ -106,7 +106,21 @@ const FeedbackModal = () => {
         });
     });
   };
+  
+function RatingComponent() {
+  const [rating, setRating] = useState(null);
```
[the remark this answers is not in the document]
Comment (codervivek5) [reply src/User/components/FeedbackForm/Feedback.jsx]: Check this

**Review by codervivek5 — APPROVED** [review]

**Review by codervivek5 — APPROVED** [review]

**Comment by Damini2004** [comment]
@codervivek5 kindly add gssoc label too.

**Pull request merged**